.clang-format

```diff
@@ -29,13 +32,18 @@ BraceWrapping:
   IndentBraces: false
   SplitEmptyFunction: false
   SplitEmptyRecord: true
+BinPackArguments: false
+BinPackParameters: true
 BreakBeforeBinaryOperators: NonAssignment
 BreakBeforeTernaryOperators: false
 BreakConstructorInitializers: BeforeColon
 BreakInheritanceList: BeforeColon
-ColumnLimit: 0
+ColumnLimit: 120
```

Comment:
I disagree. Having a column limit can help improve readability and encourages people to not put everything on one line or make a method with 20 parameters. I intentionally set it a bit high at 120 instead of the usual 80 that many companies use to allow for those longer lines. If a line really needs to be long or a specific format then the @clang-format off specifier can be used instead of sacrificing the formatting of everywhere else. Also, I would argue that if you have a line that long then maybe the issue is not the formatting but the line itself trying to do too much.